holoviews/core/data/xarray.py

```diff
@@ -191,7 +191,8 @@ def select(cls, dataset, selection_mask=None, **selection):
             if isinstance(v, set):
                 validated[k] = list(v)
             elif isinstance(v, tuple):
-                validated[k] = slice(v[0], v[1]-sys.float_info.epsilon*10)
+                upper = None if v[1] is None else v[1]-sys.float_info.epsilon*10
```

Comment:
Why ``sys.float_info.epsilon*10``? Is ``sys.float_info.epsilon`` not enough of a difference?